browser_context.go

```diff
@@ -369,7 +370,15 @@ func (b *browserContextImpl) Close(options ...BrowserContextCloseOptions) error
 	_, err = b.channel.Send("close", map[string]interface{}{
 		"reason": b.closeReason,
 	})
-	<-b.closed
+	if err != nil {
+		return err
+	}
+	timeout := b.timeoutSettings.Timeout()
+	select {
+	case <-time.After(time.Duration(timeout) * time.Millisecond):
```

Comment:
this looks different to what we do upstream: https://github.com/microsoft/playwright/blob/e31e5b1b7c7628921875c98cc98ebdfb2b53b766/packages/playwright-core/src/client/browserContext.ts#L435

Reply:
My bad, what really works here is the previous `if err != nil`.  removed with [99c5b11](https://github.com/playwright-community/playwright-go/pull/395/commits/99c5b1160f192e2f715bd92464edfaf26ceb44e3)